mysql-test/main/func_misc.test

```diff
@@ -1278,3 +1278,59 @@ DROP TABLE t1;
 --echo #
 --echo # End of 10.4 tests
 --echo #
+
+--echo #
+--echo # MDEV-4017 - GET_LOCK() with negative timeouts has strange behavior
+--echo #
+
+--echo #
+--echo # MDEV-10569 Add RELEASE_ALL_LOCKS SQL-function 
+--echo # 
+
+--echo # Test function without any locks
+SELECT RELEASE_ALL_LOCKS();
+
+--echo # Test function with one lock only
+SELECT GET_LOCK ('l1',10);
+SELECT RELEASE_ALL_LOCKS();
```

Comment:
But no space here, please be consistent. Usually we do not put spaces in this case.